pages/devfiles/[id]/index.tsx

```diff
@@ -28,9 +42,27 @@ export const getStaticProps: GetStaticProps = async (context) => {
     return devfile.name === context.params?.id
   })!)
 
+  var res2: Response
+  var devfileText:string|null =null  
+  var devfileJSON = null
+
+  if(devfile.type==='stack'){
+    res2 =  await fetch('https://registry.devfile.io/devfiles/'+devfile.name,{
```

Comment:
nit: rename it to something like devfileEndpointResp

Reply:
resolved